examples/contrib/oed/ces_gradients.py

```diff
@@ -70,11 +85,10 @@ def forward(self, y_dict, design_prototype, observation_labels, target_labels):
         s = y.log() - y1m.log()
         x = self.softplus(self.linear1(s))
         x = self.softplus(self.linear2(x))
-        rho0 = self.softplus(self.rho0(x)).squeeze(-1)
-        rho1 = self.softplus(self.rho1(x)).squeeze(-1)
-        alpha_concentration = self.softplus(self.alpha_concentration(x))
+        rho_concentration = 1e-6 + self.softplus(self.rho_concentration(x))
+        alpha_concentration = 1e-6 + self.softplus(self.alpha_concentration(x))
         slope_mu = self.slope_mu(x).squeeze(-1)
-        slope_sigma = self.softplus(self.slope_sigma(x)).squeeze(-1)
+        slope_sigma = 1e-6 + self.softplus(self.slope_sigma(x)).squeeze(-1)
```

Comment:
match neural architectures between ces and ces_gradients. Maybe even import into ces to save space